lib/norad.gi

```diff
@@ -1050,3 +1050,12 @@ local sus,ser,len,factorhom,uf,n,d,up,mran,nran,mpcgs,pcgs,pcisom,nf,ng,np,sub,
   
 end);
 
+InstallMethod( NormalizerOp,"solvable radical", IsIdenticalObj,
+  [ IsGroup and CanComputeFittingFree, IsGroup],
+  -5, # rank lower than methods for reprersentations that imply CCFF
```

Comment:
So can we remove this?
```suggestion
```

Reply:
If it makes you happier we can rank at -1. But giv en the prominence of `Normalizer` I want to make sure the method currently ends up only as fallback if no better method is known.

Another option would be to rank one higher than the generic finite group routine, but that seemed harder.